src/web/layouts/DecideLayout.tsx

```diff
@@ -19,6 +21,15 @@ type Props = {
 	NAV: NavType;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+injectGlobal`
```

Comment:
Not at all sure this is a good way to do. An alternative might be to create an `accessibilityStyles` export and include that directly in `htmlTemplate`. Requiring every component to customise focus styling doesn't seem realistic or efficient (though we should definitely look to customising focus for a lot of components as appropriate going forward).